banco-dados/scripts/inserir-dados.sql

```diff
@@ -1,4 +1,12 @@
 INSERT INTO objetivo (descricao) VALUES ('Ganhar massa'),  ('Perder peso');
 
-INSERT INTO usuario 	(nome, 	email,  senha, 	imc, objetivo_id)  VALUES  ('joao',  'joao@email.com', '123456',  22.7,	 1);
-INSERT INTO usuario  (nome,  email,  senha,  imc, objetivo_id)  VALUES  ('maria',  'maria@email.com', '654321',  27.7,	 2);
\ No newline at end of file
+INSERT INTO usuario (nome, 	email,  senha, 	imc, objetivo_id)  VALUES  ('joao',  'joao@email.com', '123456',  22.7,	 1);
+INSERT INTO usuario  (nome,  email,  senha,  imc, objetivo_id)  VALUES  ('maria',  'maria@email.com', '654321',  27.7,	 2);
+
+INSERT INTO unidade_medida  (nome)  VALUES  ('grama'), ('pitada'), ('colher');
+
+INSERT INTO refeicao  (nome, descricao, calorias, modo_preparo, objetivo_id, usuario_id)  VALUES  ('ovo frito',  'ovo com sal direto na frigideira',  107.0,	'Ligar fogão, colocar óleo em uma frigideira e levá-la ao fogo, adicionar o ovo e o sal, esperar fritar e virar para fritar o outro lado, e fim.', 1, 1);
+
+INSERT INTO ingrediente  (nome) VALUES  ('ovo'), ('sal');
+
+INSERT INTO ingrediente_refeicao  (refeicao_id, ingrediente_id, quantidade, unidade_medida_id) VALUES  (1, 1, 63.0, 1);
```

Comment:
tem ciencia do que ta rolando aqui?
traduzindo
refeicao_id -> 1 (ovo frito)
ingrediente_id -> 1 (ovo)
quantidade -> 63.0
unidade de medida_id -> 1 (grama)

seria algo como na receita de ovo frito, adiciona o ingrediente ovo, na quantidade 63 gramas, para algo mais veridico eu faria: 
```suggestion
INSERT INTO unidade_medida (nome) VALUES ('grama'), ('pitada'), ('colher'), ('unidade'), ('pitada');

INSERT INTO refeicao  (nome, descricao, calorias, modo_preparo, objetivo_id, usuario_id)  VALUES  ('ovo frito',  'ovo com sal direto na frigideira',  107.0,	'Ligar fogão, colocar óleo em uma frigideira e levá-la ao fogo, adicionar o ovo e o sal, esperar fritar e virar para fritar o outro lado, e fim.', 1, 1);

INSERT INTO ingrediente  (nome) VALUES  ('ovo'), ('sal');

INSERT INTO ingrediente_refeicao (refeicao_id, ingrediente_id, quantidade, unidade_medida_id) VALUES (1, 1, 1, 4);
INSERT INTO ingrediente_refeicao (refeicao_id, ingrediente_id, quantidade, unidade_medida_id) VALUES (1, 2, 1, 5);
```

seria algo como:
na receita do ovo frito adicione uma unidade de ovo
na receita do ovo frito adicione uma pitada de sal